readme.md

```diff
@@ -41,7 +41,8 @@ var theBody = document.querySelectorAll("body");
 theBody.style.backgroundColor = "red";
 ```
 
-> Your answer...
+FTFY: var theBody = document.getElementsByTagName("BODY")[0];
+      theBody.style.backgroundColor = "red";
```

Comment:
:+1: only i think it does need to be lower case. But the idea is `querySelectorAll` and `getElementsByTagName` return an array of elements therefore need to be indexed liek you did